precompiles/randomness/Randomness.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.0;
+
+interface Randomness {
+    /// @dev Interface for all randomness consumers
```

Comment:
This doc comment should be above the `interface` line.

Reply:
Also this is not the interface for randomness consumers. It is the interface of the randomness precompile.
There should be another .sol file with the interface each randomness consumer should implement?